crates/wasmtime/src/store.rs

```diff
@@ -1166,17 +1164,87 @@ impl StoreOpaque {
         unsafe { wasmtime_runtime::gc(&self.modules, &mut self.externref_activations_table) }
     }
 
-    pub fn lookup_trampoline(&self, anyfunc: &VMCallerCheckedAnyfunc) -> VMTrampoline {
-        // Look up the trampoline with the store's trampolines (from `Func`).
+    /// Looks up the corresponding `VMTrampoline` which can be used to enter
+    /// wasm  given an anyfunc function pointer.
+    ///
+    /// This is a somewhat complicated implementation at this time unfortnately.
+    /// Trampolines are a sort of side-channel of information which is
+    /// specifically juggled by the `wasmtime` crate in a careful fashion. The
+    /// sources for trampolines are:
+    ///
+    /// * Compiled modules - each compiled module has a trampoline for all
+    ///   signatures of functions that escape the module (e.g. exports)
```

Comment:
```suggestion
    ///   signatures of functions that escape the module (e.g. exports and `ref.func`able functions)
```

Right? Just double checking my understanding.

Reply:
Indeed!